codelists/sources.csv

```diff
@@ -0,0 +1,4 @@
+Code,Title,Description
+epo-notice,Notice,Notice
+epo-procurement-document,Procurement Document,Procurement document
+epo-sub-espd,European Single Procurement Document (ESPD),European single procurement document
```

Comment:
Let's keep it identical, since the description is not otherwise adding anything to the title.

```suggestion
epo-procurement-document,Procurement Document,Procurement Document
epo-sub-espd,European Single Procurement Document (ESPD),European Single Procurement Document (ESPD)
```